src/main.ts

```diff
@@ -19,6 +19,10 @@ import { environment } from './environments/environment';
 
 if (environment.production) {
   enableProdMode();
+
+  if (window) {
+    window.console.log = function() {};
+  }
```

Comment:
What is this for?

Reply:
for the avoid console log printing in production mode.

Reply:
is there an option for debug or you simply removed the printout?